src/backend/src/routes/reimbursement-requests.routes.ts

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import { body } from 'express-validator';
+import { intMinZero, isAccount, isDate, nonEmptyString } from '../utils/validation.utils';
+import { validateInputs } from '../utils/utils';
+import ReimbursementRequestController from '../controllers/reimbursement-requests.controllers';
+
+const reimbursementRequestsRouter = express.Router();
+
+reimbursementRequestsRouter.post(
+  '/create',
+  isDate(body('dateOfExpense')),
+  nonEmptyString(body('vendorId')),
+  isAccount(body('account')),
+  body('receiptPictures').isArray(),
+  nonEmptyString(body('receiptPictures.*')),
+  nonEmptyString(body('reimbursementProducts.*.name')),
```

Comment:
you should verify reimbursementProducts is an array. Right now you can forget to pass it in the body, and the endpoint doesn't complain but it throws an uncaught exception in the backend when you try to validate the reimbursement products. 

Also, should we allow a reimbursement request to be submitted that has no associated reimbursement products? (aka an empty array for reimbursementProducts), probably not right?